### ISSUE TITLE
[MU4 Issue] Numbers should be added to same instruments on replace instrument

### ISSUE THREAD

**Issue by HemantAntony** [musescore/MuseScore#10123]
**Describe the bug**
When you replace an instrument with another already existing instrument, numbers should be added in the instruments panel

**To Reproduce**
1. Create a score with a couple of instruments (For this example it must have a Flute)
2. Replace an instrument say, Oboe with Flute, so there are two Flutes now
3. The flutes are displayed just as "Flute" and "Flute"

**Expected behavior**
The flutes have to be numbered as 1 and 2.

**Screenshots**

https://user-images.githubusercontent.com/88136054/146491863-bbcbd74a-c186-43ea-867f-9ca57977d418.mp4

**Desktop (please complete the following information):**
 - OS: Windows 11

**Comment by HemantAntony** [comment]
A silly question here: in my mind, the existing flute should be called Flute 1 and the new flute should be called Flute 2. Do you agree with this?

**Comment by njvdberg** [comment]
When creating a new score with multi flutes, it is fine to number the parts `Flute 1`, `Flute 2` etc.

But if you add a flute afterwards, or replace an existing instrument, how can you tell which should `Flute 1` and `Flute 2`. Suppose I have piccolo and a flute, replace the piccolo to a flute will most likely result in `Flute 1` (the old piccolo part and `Flute 2` the old flute part. But when you have a score with flute and an oboe, replace the oboe by a flute will most likely result into a `Flute 1` (the original flute part) and a `Flute 2` (the old oboe part).
So, the original flute part can become a `Flute 1` or a `Flute 2`, depending on the context.

And what should happen if you have already a `Flute 1` and `Flute 2`, in the first case, with the piccolo, they will be renumbered to `Flute 2` and  `Flute 3`? And what if the parts are already named `1e Fluit` and `2e Fluit` (dutch), how will they be renamed?

Altogether, I'm not sure renumbering is a good idea.

**Comment by HemantAntony** [comment]
Yeah, I think you make a good point. Plus, I just now realized that the user themself can change the instrument name. So I will close this issue. Thanks!
